storage/simulation/cluster.go

```diff
@@ -223,45 +223,90 @@ func (c *Cluster) prepareActions() {
 
 // performActions performs a single action, if required, for each range.
 func (c *Cluster) performActions() {
-	// TODO(Bram): convert this to run a single action per store instead of a
-	// single action per range.
-	for _, rangeID := range c.rangeIDs {
-		r := c.ranges[rangeID]
-		nextAction, rebalance := r.getNextAction()
-		switch nextAction {
-		case storage.AllocatorAdd:
-			newStoreID, err := r.getAllocateTarget()
-			if err != nil {
-				fmt.Fprintf(c.actionWriter, "Error: %s\n", err)
-				continue
+	tw := tabwriter.NewWriter(c.actionWriter, 12, 1, 2, ' ', 0)
```

Comment:
I'm going to convert everything to using a single tabwriter and add a flush function on the cluster as whole.